api/v3/Case.php

```diff
@@ -742,6 +742,10 @@ function civicrm_api3_case_getList($params) {
     $params['params']['case_id'] = ['IN' => $params['id']];
     unset($params['id']);
   }
+  else {
+    $params['params']['case_id.is_deleted'] = 0;
+    $params['params']['contact_id.is_deleted'] = 0;
```

Comment:
Should these be `if (!isset($params[XXX]) ...`, since otherwise there's no way to call getlist to get deleted cases.

On a general note, I guess this hasn't come up much because the main place it's used [already filters deleted](https://github.com/civicrm/civicrm-core/blob/9d222550785db570a3e09a620d62233a77f38c17/CRM/Case/Form/ActivityToCase.php#L78).

Reply:
Oops wrong link ... fixed.